datafusion/sql/tests/integration_test.rs

```diff
@@ -199,6 +199,17 @@ fn cast_to_invalid_decimal_type() {
     }
 }
 
+#[test]
+fn plan_create_table_with_pk() {
+    let sql = "create table person (id int, name string, primary key(id))";
+    let plan = r#"
+CreateMemoryTable: Bare { table: "person" }
```

Comment:
I'd like to change this to `CreateMemoryTable: Bare { table: "person", primary_key="x, y" }`, but I couldn't figure out where `Debug` is `impl`ed for `CreateMemoryTable`.

Reply:
I think it's https://github.com/apache/arrow-datafusion/blob/main/datafusion/expr/src/logical_plan/plan.rs#L1051
and 
https://github.com/apache/arrow-datafusion/blob/main/datafusion/common/src/table_reference.rs#L110

Reply:
That worked, ty!